frontend/app/.server/locales/protected-en.json

```diff
@@ -35,5 +35,12 @@
   "privacy-statement": {
     "confirm-privacy-notice-checkbox": "I agree to the terms and conditions",
     "page-title": "Privacy Statement"
+  },
+  "primary-identity-document": {
+    "current-status-in-canada": {
+      "title": "Current status in Canada",
+      "option-1": "Canadian citizen born outside Canada"
```

Comment:
Might be considered a nitpick, but I think the i18n key for this should be something more descriptive. `option-outside-canada` for example.

You can even do something like:

``` json
"options": {
  "outside-canada": "...",
  "inside-canada": "...",
  "underneath-canada": "...",
  "behind-canada": "...",
}
```

It's not inconceivable that the order of these options could be changed over time. And it also makes it slightly easier to read the code when the keys are more descriptive.